src/app/features/courses/courses.component.ts

```diff
@@ -13,58 +14,60 @@ import { Subject, takeUntil } from 'rxjs';
 })
 export class CoursesComponent implements OnInit, OnDestroy {
   private destroy$: Subject<boolean> = new Subject();
-  courses!: Course[];
-  selectedCourse!: Course | null;
+  courses$ = this.coursesFacade.allCourses$;
+  selectedCourse$: Observable<Course | null>;
+  course!: Course | null;
```

Comment:
What purpose does this class member serve?

Reply:
well it was used by the onClickAddCourse method but i think it is completely unneccesary so you're right, deleted it